package.json

```diff
@@ -8,7 +8,7 @@
 	],
 	"scripts": {
 		"commit": "cz",
-		"build": "lerna exec --stream 'npm run build'",
+		"build": "lerna run build",
 		"build:docs": "lerna exec --stream --parallel 'npm run build:docs' && npm run build:feed",
```

Comment:
Do you think here we can also replace 
```lerna exec --stream --parallel 'npm run build:docs' && npm run build:feed``` 
with 
```lerna run build:docs && npm run build:feed```

As well as for `dev`, `lint` and `format`?

Reply:
That should not have been in this PR - sorry, wrong base selected...

But... I don't think that the build:docs command will benefit much from any of the caching though...